product-updates/knowledge-center/index.html

```diff
@@ -329,6 +333,61 @@ <h2 class="usa-card__heading">Use an existing Login.gov Account</h2>
+            </div>
+            <div class="usa-icon-list__content">
+              <h5 class="usa-icon-list__title">
+                Additions to knowledge center FAQs
+              </h5>
+              <p>
+                The knowledge center now offers new guidance including where to go for help if you encounter an unexpectedly high volume of errors in your data, how to proceed in a situation where you cannot correct a certain type of error, and how to utilize resubmission to improve your data quality and completeness throughout a fiscal quarter.
+              </p>
+            </div>
+          </li>
+        </ul>
+
+          <h4 style="margin-top: 16px; color: #595959;">Changed / Fixed:</h4>
+        <ul class="usa-icon-list">
+
+    <li class="usa-icon-list__item" style="margin-top: 16px;">
+      <div class="usa-icon-list__icon text-ink">
+          <img src="../img/changed.svg" aria-hidden="true" style="background-color: #b3e2ff; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+      </div>
+      <div class="usa-icon-list__content">
+        <h5 class="usa-icon-list__title">
+          Increased actionability of file upload error messages
+        </h5>
+        <p>
+      In cases where a user attempts to upload an incompatible file type to TDP we're now triggering a more <a href="lightbox">descriptive error message</a> that describes which file types are accepted and compatible with TDP; Files ending in .txt, .ms##, .ts##, or .ts###. The filename of the incompatible file also now stays onscreen so that it can be more easily compared to the list of accepted file types. 
+        </p>
+      </div>
+    </li>
+
+  </ul>
+
+   <h4 style="margin-top: 16px; color: #595959;">In Development:</h4>
+      <ul class="usa-icon-list">
+
+  <li class="usa-icon-list__item" style="margin-top: 16px;">
+    <div class="usa-icon-list__icon text-ink">
+      <img src="../img/in-dev.svg" aria-hidden="true" style="background-color: #fff4bd; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+    </div>
+    <div class="usa-icon-list__content">
+      <h5 class="usa-icon-list__title">
+        Expanded file processing and submission history features
+      </h5>
+      <p>
+        Building on work that began in v 3.1, TDP will soon feature more advanced submission file processing able to serve up detailed information about the data quality of newly uploaded files. This includes comparison of TANF cases with errors to error-free TANF cases within the file as well as an overall status for the file describing different levels of data quality on the road to error-free and fully accepted. We will keep you updated on the progress of these features and communicate when it will become available for use and benefit.
```

Comment:
can we use something different than "error-free" since this isnt necessarily true? cases with no errors detected by the system, or something like that.